app/src/main/res/values/strings.xml

```diff
@@ -251,49 +251,49 @@
     <string name="count_milliseconds_short">%d ms</string>
     <string name="use_recommended_fonts">Apply the Recommended Font</string>
     <string name="use_recommended_fonts_notice">Use the supplied custom font</string>
-    <string name="waiting">Waiting</string>
-    <string name="theme_optimize">Optimize for AMOLED Screens</string>
-    <string name="theme_optimize_summary">Use darker colors</string>
+    <string name="waiting">Waiting…</string>
+    <string name="theme_optimize">AMOLED Optimization</string>
```

Comment:
I am not sure about this tbh. I know the older one was not good, but doesn't this also erase the context?

![Screenshot_20210828-161134_TShot_DEBUG](https://user-images.githubusercontent.com/10258162/131219070-54f77a45-b046-4c80-b71c-e5691ab947b8.png)

Reply:
If the context is that darker colours look better on AMOLED, then "dark" would be it.
Pure black doesn't look better because of contrast bleeding in the sensory realm, so I thought the point to make
was that of using less power.

Reply:
Yes, the point is mainly about using less power, however, AMOLED Optimization doesn't immediately give away that the change will be related to theming. Maybe **AMOLED Display Optimization** could be a better fit?